packages/typespec-test/test/authoring/generated/openapi/2022-05-15-preview/openapi.json

```diff
@@ -929,7 +929,7 @@
     },
     "Azure.Core.Foundations.InnerError": {
       "type": "object",
-      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors.",
+      "description": "An object containing more specific information about the error. As per Azure REST API guidelines - https://aka.ms/AzureRestApiGuidelines#handling-errors.",
```

Comment:
refer to : https://github.com/Azure/typespec-azure/pull/2599/files#diff-8e0e35d9ca28fb3e2b7795cda2cf6753eb3cd6f29267ec2b8864ad5f38c502caR167